heartbeat/scheduler/scheduler.go

```diff
@@ -23,20 +23,27 @@ import (
 	"sync"
 	"time"
 
+	batomic "github.com/elastic/beats/libbeat/common/atomic"
```

Comment:
Was there a conflict just using atomic?

Reply:
looks like `sync/atomic` was still imported while this import was added. grepping new `scheduler.go` there should be no more naming conflict by now.